examples/pylintrc

```diff
@@ -11,25 +28,39 @@ extension-pkg-allow-list=
 # for backward compatibility.)
 extension-pkg-whitelist=
 
+# Return non-zero exit code if any of these messages/categories are detected,
+# even if score is above --fail-under value. Syntax same as enable. Messages
+# specified are enabled, while categories only check already-enabled messages.
+fail-on=
+
 # Specify a score threshold to be exceeded before program exits with error.
 fail-under=10
 
+# Interpret the stdin as a python script, whose filename needs to be passed as
+# the module_or_package argument.
+from-stdin=False
+
 # Files or directories to be skipped. They should be base names, not paths.
 ignore=CVS
 
+# Add files or directories matching the regex patterns to the ignore-list. The
+# regex matches against paths and can be in Posix or Windows format.
+ignore-paths=
+
 # Files or directories matching the regex patterns are skipped. The regex
-# matches against base names, not paths.
-ignore-patterns=
+# matches against base names, not paths. The default value ignores Emacs file
+# locks
+ignore-patterns=^\.#
 
-# Add files or directories matching the regex patterns to the ignore-list.
-# The regex matches against paths. NOTE: Because "\" represents the directory
-# delimiter on Windows systems it can't be used as the escape character
-# in these regular expressions.
-ignore-paths=
+# List of module names for which member attributes should not be checked
+# (useful for modules/projects where namespaces are manipulated during runtime
+# and thus existing member attributes cannot be deduced by static analysis). It
+# supports qualified module names, as well as Unix pattern matching.
+ignored-modules=
 
 # Python code to execute, usually for sys.path manipulation such as
 # pygtk.require().
-#init-hook=
+init-hook=None
```

Comment:
Here in particular, I think we expect a string that will be executed. None is not wrong but it's misleading.

Reply:
I think ``--generate-rcfile`` might not handle the `pre-processed` options and their default values very well.

I handled those for ``--generate-toml-config`` but that might be broken for `rcfile`?